src/main/java/org/truffleruby/core/binding/BindingNodes.java

```diff
@@ -395,4 +401,20 @@ public DynamicObject allocate(DynamicObject rubyClass) {
 
     }
 
+    @Primitive(name = "caller_binding")
```

Comment:
Fixed in 482030a.
The error now looks like this:
```
[ruby] WARNING node org.truffleruby.core.binding.BindingNodes.CallerBindingNode should have needsSelf = false
```
I also added the needsSelf.